### ISSUE TITLE
Bump tar version (solve 2 high severity vulnerabilities)

### ISSUE THREAD

**Issue by bbailleux** [mapbox/node-pre-gyp#597]
The `tar` module used in `node-pre-gyp`, currently version **6.1.0**, has _two_ recently published high severity vulnerabilities.

See:

* [Arbitrary File Creation/Overwrite due to insufficient absolute path sanitization](https://www.npmjs.com/advisories/1770)
* [Arbitrary File Creation/Overwrite via insufficient symlink protection due to directory cache poisoning](https://www.npmjs.com/advisories/1771)

Those vulnerabilities has been fixed in `tar` version **6.1.2** (current version is **6.1.6**)

Would you please consider upgrading (those CVEs break our CI/CD)?

**Comment by QuteBits** [comment]
Agreed, some of our production servers started spewing those 2 in the logs recently. Also went through the last 20 releases of `node-pre-gyp` and didn't find any that would not have those 2 errors, so currently there is no release to fall back on.

**Comment by jeksmith** [comment]
Please do this ASAP. Need tar version 6.1.9 now. Latest vulnerabilities:

- [Arbitrary File Creation/Overwrite via insufficient symlink protection due to directory cache poisoning using symbolic links](https://npmjs.com/advisories/1780)
- [Arbitrary File Creation/Overwrite on Windows via insufficient relative path sanitization](https://npmjs.com/advisories/1781)

**Comment by jeksmith** [comment]
Actually there need not be a version bump afterall. node-pre-gyp defines the tar dependency as "tar": "^6.1.0", which allows for an update to 6.1.9.
